### ISSUE TITLE
Crash report 7.4-7.2.1: RuntimeException from TransactionTooLargeException in ActivityLauncher.editPostOrPageForResult

### ISSUE THREAD

**Issue by theck13** [wordpress-mobile/WordPress-Android#5974]
```
Fatal Exception: java.lang.RuntimeException: Failure from system
       at android.app.Instrumentation.execStartActivity(Instrumentation.java:1514)
       at android.app.Activity.startActivityForResult(Activity.java:3930)
       at android.support.v4.app.BaseFragmentActivityJB.startActivityForResult(BaseFragmentActivityJB.java:50)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:79)
       at android.app.Activity.startActivityForResult(Activity.java:3890)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:859)
       at org.wordpress.android.ui.ActivityLauncher.editPostOrPageForResult(ActivityLauncher.java:197)
       at org.wordpress.android.ui.posts.PostsListFragment.onPostButtonClicked(PostsListFragment.java:469)
       at org.wordpress.android.ui.posts.adapters.PostsListAdapter$4.onClick(PostsListAdapter.java:479)
       at android.view.View.performClick(View.java:5204)
       at android.view.View$PerformClick.run(View.java:21153)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5417)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by android.os.TransactionTooLargeException: data parcel size 1282428 bytes
       at android.os.BinderProxy.transactNative(Binder.java)
       at android.os.BinderProxy.transact(Binder.java:503)
       at android.app.ActivityManagerProxy.startActivity(ActivityManagerNative.java:2657)
       at android.app.Instrumentation.execStartActivity(Instrumentation.java:1507)
       at android.app.Activity.startActivityForResult(Activity.java:3930)
       at android.support.v4.app.BaseFragmentActivityJB.startActivityForResult(BaseFragmentActivityJB.java:50)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:79)
       at android.app.Activity.startActivityForResult(Activity.java:3890)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:859)
       at org.wordpress.android.ui.ActivityLauncher.editPostOrPageForResult(ActivityLauncher.java:197)
       at org.wordpress.android.ui.posts.PostsListFragment.onPostButtonClicked(PostsListFragment.java:469)
       at org.wordpress.android.ui.posts.adapters.PostsListAdapter$4.onClick(PostsListAdapter.java:479)
       at android.view.View.performClick(View.java:5204)
       at android.view.View$PerformClick.run(View.java:21153)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5417)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by maxme** [comment]
Should be fixed by https://github.com/wordpress-mobile/WordPress-Android/pull/5856

**Comment by oguzkocer** [comment]
This is hopefully fixed in #5856
